bucket/simple-dnscrypt.json

```
@@ -0,0 +1,62 @@
+{
+    "version": "0.7.1",
+    "description": "An easy to use GUI for dnscrypt-proxy",
+    "homepage": "https://simplednscrypt.org/",
+    "license": "MIT",
+    "suggest": {
+        "vcredist2017": "extras/vcredist2017"
```

```suggestion
        "vcredist2015-2022": "extras/vcredist2022"
```
[VCRedist2022 now covers 2015-2022](https://docs.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170)